__tests__/app.test.js

```diff
@@ -50,3 +50,44 @@ describe("/api", () => {
     
   });
 });
+
+describe("/api/articles/:article_id", () => {
+  test('GET: 200 sends a single article to the client', () => {
+    return request(app).get('/api/articles/1').expect(200)
+    .then(({body}) => {
+      const {article} = body;
+      console.log(body)
+
+      expect(typeof article.author).toBe('string')
+      expect(typeof article.title).toBe('string')
+      expect(typeof article.article_id).toBe('number')
+      expect(typeof article.body).toBe('string')
+      expect(typeof article.topic).toBe('string')
+      expect(typeof article.created_at).toBe('string')
+      expect(typeof article.votes).toBe('number')
+      expect(typeof article.article_img_url).toBe('string')
+      
+      expect(article.author).toBe('butter_bridge')
+      expect(article.title).toBe('Living in the shadow of a great man')
+      expect(article.article_id).toBe(1)
```

Comment:
It would be enough just to check the specific id here, rather than checking the values of all properties.